HW-11/HW12/Homework_12.py

```diff
@@ -0,0 +1,71 @@
+import json
+from datetime import datetime
+
+# 1
+with open('phonebook.txt', 'r+', encoding='utf-8') as phone1:
+    try:
+        phone_book = json.load(phone1)
+    except Exception:
+        phone_book = []
```

Comment:
In your original phonebook you use dict instead of list. I think a dict is much better choice